session/session_starter.py

```diff
@@ -132,7 +147,18 @@ def start_multivac_session(environment_name, agent_name, num_steps, observation_
     :param video_fps: Frame per second of the output recording of the gym environment.
     :param display_video: Flag to determine whether or not to manually display session in a window separate from the
                           device/emulator or UI.
+    :return SessionStatusEnum indicating how the session concluded.
     """
+
+    # Input validation
+    assert type(environment_name) == str  # Other validation for environment name is done by Multivac
+    assert type(agent_name) == str  # Other validation for agent name is done by Multivac
+    assert type(num_steps) == int and num_steps > 0, "Specify an integer num_steps greater than zero"
```

Comment:
Designate max num_steps and observation_delta and communicate these to the frontend and command line